appinfo/info.xml

```diff
@@ -11,14 +11,13 @@
 		This sets the type of activity to record, as well as whether to the user sees their own activities,
 		whether these are only available online, and whether they get an email digest on a regular basis.
 		More information is available in the Activity documentation.
-	</description>
+</description>
 
 	<version>2.16.0</version>
 	<licence>agpl</licence>
 	<author>Frank Karlitschek</author>
 	<author>Joas Schilling</author>
 
-	<default_enable />
```

Comment:
This needs to stay, downloading the correct `.xsd` will fix it

Reply:
Thanks for the pointer :+1: